test/ping_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require "test_helper"
+
+class PingControllerTest < RailsTwirp::IntegrationTest
+  test "you can ping it" do
+    req = RPC::DummyAPI::PingRequest.new(name: "Bouke")
+    rpc RPC::DummyAPI::DummyService, "Ping", req
+    assert_equal "BoukeBouke", response.double_name
+  end
+
+  test "you can ping render" do
+    req = RPC::DummyAPI::PingRequest.new(name: "Bouke")
+    rpc RPC::DummyAPI::DummyService, "PingRender", req
+    refute_instance_of Twirp::Error, response
+    assert_equal "BoukeBouke", response.double_name
+  end
+
+  test "you can ping template" do
+    req = RPC::DummyAPI::PingRequest.new(name: "Bouke")
+    rpc RPC::DummyAPI::DummyService, "PingTemplate", req
+    refute_instance_of Twirp::Error, response
+    assert_equal "BoukeBouke", response.double_name
+  end
+
+  test "error response" do
+    req = RPC::DummyAPI::PingRequest.new
+    rpc RPC::DummyAPI::DummyService, "ErrorResponse", req
+    assert_instance_of Twirp::Error, response
+    assert_equal "You are not authenticated!!", response.msg
+    assert_equal :unauthenticated, response.code
+  end
+
+  test "raise error" do
+    req = RPC::DummyAPI::PingRequest.new
+    rpc RPC::DummyAPI::DummyService, "RaiseError", req
+    assert_instance_of Twirp::Error, response
+    assert_equal "Not found", response.msg
+    assert_equal :not_found, response.code
+  end
+
+  test "uncaught error" do
+    req = RPC::DummyAPI::PingRequest.new
+    rpc RPC::DummyAPI::DummyService, "UncaughtError", req
+    assert_instance_of Twirp::Error, response
+    assert_equal "Uncaught", response.msg
```

Comment:
We may want to add some logging for this case (that integrates with the Rails logger)

Reply:
https://github.com/cheddar-me/rails-twirp/issues/2 :+1: